File: gerenciador de tarefas 1.0/controllers/Gerenciador.php
<?php

class Gerenciador {
    private $usuarios;

    public function __construct(){
        $this->usuarios = array();
    }

    public function validarDados($nome, $senha) {
        foreach ($this->usuarios as $usuario) {
            if ($nome === $usuario->getNome() && $senha === $usuario->getSenha()) {
                return true;
            }
        }
        return false;
    }

    public function adicionarUsuario($usuario){
        $this->usuarios[] = $usuario;
    }

    public function getUsuario($nomeUsuario) {
        foreach ($this->usuarios as $usuario) {
            if ($usuario->getNome() === $nomeUsuario) {
                return $usuario;
            }
        }
        return null;
    }
}


?>

File: gerenciador-de-tarefas-spring/gerenciador-de-tarefas/src/main/java/com/tanyrm/gerenciadordetarefas/controller/Gerenciador.java
package controller;

import model.Usuario;

import java.util.ArrayList;

public class Gerenciador {
    private ArrayList<Usuario> usuarios;

    public Gerenciador() {
        this.usuarios = new ArrayList<Usuario>();
    }

    public boolean validarDados(String nome, String senha) {
        for (Usuario usuario : usuarios){
            if ((nome.equals(usuario.getNome())) && (senha.equals(usuario.getSenha()))) {
                return true;
            }
        }
        return false;
    }

    public void adicionarUsuario(Usuario usuario) {
        this.usuarios.add(usuario);
    }

    public Usuario getUsuario(String nome) {
        for (Usuario usuario : usuarios) {
            if (usuario.getNome().equals(nome)) {
                return usuario;
            }
        }
        return null;
    }
}
